app.js

```diff
@@ -1,134 +1,117 @@
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
-
+var storeArray = [];
 var firstPike = new Store('1st and Pike', 23, 65, 6.3);
 var seaTac = new Store('SeaTac Airport', 3, 24, 1.3);
 var seaCenter = new Store('Seattle Center', 11, 38, 2.3);
 var capHill = new Store('Capitol Hill', 20, 38, 2.3);
 var alki = new Store('Alki', 2, 16, 4.6);
 
-//store object:
 function Store(name, minCust, maxCust, avgCookieSale) {
   this.name = name;
   this.minCust = minCust;
   this.maxCust = maxCust;
   this.avgCookieSale = avgCookieSale;
   this.total = 0;
   this.salesArray = [];
-  var salesArray = this.salesArray;
   var total = this.total;
-
   this.randomCust = function() {
     return Math.floor(Math.random() * (maxCust - minCust + 1) + minCust);
   };
-
   this.randomHourlySale = function() {
     return Math.floor(this.randomCust() * avgCookieSale);
   };
-
   this.totalSales = function() {
     for (var i = 0; i < storeHours.length; i++) {
       var randomHourlySale = this.randomHourlySale();
-      salesArray.push(randomHourlySale);
+      this.salesArray.push(randomHourlySale);
```

Comment:
Something to consider: every time this method is called, this line will push new data into the `salesArray`. That means that if it's called twice for whatever reason, your array will have double the amount of data (so instead of being length 14 it'll be length 28). Consider how you might combat this.